### ISSUE TITLE
Searching for MinecraftImpl can prevent launch on some versions.

### ISSUE THREAD

**Issue by codieradical** [codieradical/MineOnline#83]
Black screens.

```
Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class np
	at on.<init>(SourceFile:14)
	at net.minecraft.client.Minecraft.<init>(SourceFile:168)
	at l.<init>(SourceFile:34)
	at net.minecraft.client.MinecraftApplet.init(SourceFile:34)
	at gg.codie.mineonline.MinecraftLauncher.startMinecraft(MinecraftLauncher.java:669)
	at gg.codie.mineonline.gui.MainMenuScreen$1.onClick(MainMenuScreen.java:49)
	at gg.codie.mineonline.gui.components.MediumButton.update(MediumButton.java:93)
	at gg.codie.mineonline.gui.MainMenuScreen.update(MainMenuScreen.java:113)
	at gg.codie.mineonline.gui.MenuManager.main(MenuManager.java:183)
	at gg.codie.mineonline.MineOnline.main(MineOnline.java:14)
```

**Comment by codieradical** [comment]
Testing more versions

- b1.9-pre4
- b1.9-pre5

**Comment by codieradical** [comment]
Fixed in 1.0.6_01
